project_task_delegate/models/project.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    delegated_user_id = fields.Many2one(
+        'res.users', related='child_ids.user_id', string='Delegated To')
+
+    @api.model
+    def _delegate_task_attachments(self, delegated_task_id):
+        attachment = self.env['ir.attachment']
+        attachment_ids = attachment.search(
+            [('res_model', '=', self._name), ('res_id', '=', self.id)])
+        new_attachment_ids = []
+        for attachment_id in attachment_ids:
+            new_attachment_ids.append(attachment_id.copy(
+                default={'res_id': delegated_task_id}))
```

Comment:
Ideally you would just unlink the attachments.
But I understand that doing so is not so straightforward.
We can leave it as is, but could you add a note on the Roadmap as a possible improvement?